hangman.rb

```diff
@@ -0,0 +1,262 @@
+# Pair: Alice Rhomieux
+require 'colorize'
+
+class Hangman
+  def initialize
+    @game_word = pick_word
+    @wrong_letters = []
+    @errors = 0
+
+    # initializes array_of_slots with underscores
+    @num_of_slots = @game_word.length
+    @array_of_slots = []
+    @num_of_slots.times do
+      @array_of_slots.push("_ ")
```

Comment:
You can also use `@array_of_slots = Array.new(@num_of_slots, "_ ")` which will initialize an array with the default number of values that are specified.